website/src/util/RoadmapItems.ts

```diff
@@ -0,0 +1,192 @@
+/*****************************************************************/
+/**                                                              */
+/** FORMAT:
+ *  {
+ *    title: "Title",
+ *    text: "Description",
+ *    source: "Source",
+ *    link: "Link to source",
+ *  }
```

Comment:
Easier to understand with an example?
```suggestion
/** FORMAT EXAMPLE:
 *  {
 *    title: "Enghouse DMS",
 *    text: "Sends the right handset settings to newly ported subscriptions",
 *    source: "Enghouse",
 *    link: "https://www.enghouse.com/",
 * },
```